web/src/main/java/pl/edu/agh/papaya/rest/sprint/service/SprintService.java

```diff
@@ -0,0 +1,70 @@
+package pl.edu.agh.papaya.rest.sprint.service;
+
+import java.time.LocalDateTime;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.function.Function;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.edu.agh.papaya.api.model.SprintState;
+import pl.edu.agh.papaya.model.Sprint;
+import pl.edu.agh.papaya.repository.SprintRepository;
+
+@Service
+@SuppressWarnings({"PMD.BeanMembersShouldSerialize"})
+public class SprintService {
+
+    private static final EnumMap<SprintState, Function<LocalDateTime, List<Sprint>>> ACCESS_METHODS =
```

Comment:
nie widzę sensu, żeby to było statyczne, jeśli jest wypełniane w konstruktorze

Reply:
Racja, pozostałość po testach z innym podejściem